tmp/.gitignore

```diff
@@ -0,0 +1,3 @@
+src/oa
+src/schema
+lib/
```

Comment:
I'll remove these files from the `.gitignore` when moving this stuff over to models-ts